Move case features to the correct section in kibana release notes (#157805)
elastic/kibana: docs/CHANGELOG.asciidoc

* Adds table tabs showing summary of metrics {kibana-pull}153044[#153044]
* Adds warning to Edit Rule Flyout when publicUrl is not configured {kibana-pull}149832[#149832]

Cases::

> * Adds support for file attachments in Cases {kibana-pull}154436[#154436]

Reviewer: Could you move the following into the Cases section from the Alerting section:

- Adds the Cases column to the alerts table [#150963](https://github.com/elastic/kibana/pull/150963)
- Adds filtering and sorting for the case activity [#149396](https://github.com/elastic/kibana/pull/149396)
- Adds the ability to filter user activities with pagination [#152702](https://github.com/elastic/kibana/pull/152702)

* Adds pagination to source map API {kibana-pull}145959[#145959]
* Adds ability to offset point labels on maps {kibana-pull}145773[#145773]

Cases::
* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]
* Redesign all cases list select modal {kibana-pull}149851[#149851]

> * RenderCustomActionsRow with named params instead of args {kibana-pull}149304[#149304]

Reviewer: Thanks for the changes! This should not be part of the Cases section.

Author: Oops! Good catch - I moved 149304 back to alerting.

Addressed in [a5d7c8d](https://github.com/elastic/kibana/pull/157805/commits/a5d7c8dd176cb1e880ae7b6e318b68a5084c3347).

* Adds pagination to source map API {kibana-pull}145959[#145959]
* Adds ability to offset point labels on maps {kibana-pull}145773[#145773]

Cases::
* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]

> * Redesign all cases list select modal {kibana-pull}149851[#149851]

Reviewer 2: ```suggestion
* Improves the design of all cases list select modal {kibana-pull}149851[#149851]
```

> * Users can now click a button on Case Detail and All Cases List to copy a case's UUID to the clipboard {kibana-pull}148962[#148962]

Reviewer 2: ```suggestion
* Adds a button to Case Detail and All Cases List to copy case UUIDs to the clipboard {kibana-pull}148962[#148962]
```

> * Sorting, severity filter and status filter are now persisted in the URL and local storage for the all cases list {kibana-pull}148549[#148549]

Reviewer 2: ```suggestion
* Adds the ability to persist sorting, severity filter, and status filter in the URL and local storage for the all cases list {kibana-pull}148549[#148549]
```

> * Sorting by status, severity, and title is now allowed in the all-cases list {kibana-pull}148193[#148193]

Reviewer 2: ```suggestion
* Adds the ability to allow sorting by status, severity, and title in the all-cases list {kibana-pull}148193[#148193]
```

> * See "My recently assigned cases" to the recent cases widget of Security Solution dashboard {kibana-pull}147763[#147763]

Reviewer 2: ```suggestion
* Adds "My recently assigned cases" to the recent cases widget of the Security Solution dashboard {kibana-pull}147763[#147763]
```

Author: ```suggestion
* See "My recently assigned cases" to the recent cases widget of Overview dashboard in Security {kibana-pull}147763[#147763]
```

* Adds the ability to bulk edit assignees on multiple cases {kibana-pull}146907[#146907]

> * Save draft user comment {kibana-pull}146327[#146327]

Reviewer 2: ```suggestion
* Adds the ability to save draft comments {kibana-pull}146327[#146327]
```

* Adds pagination to source map API {kibana-pull}145959[#145959]
* Adds ability to offset point labels on maps {kibana-pull}145773[#145773]

Cases::

> * Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]

Reviewer: Could you move the following into the Cases section from the Alerting section:

- Redesign all cases list select modal [#149851](https://github.com/elastic/kibana/pull/149851)
- Users can now click a button on Case Detail and All Cases List to copy a case’s UUID to the clipboard [#148962](https://github.com/elastic/kibana/pull/148962)
- Sorting, severity filter and status filter are now persisted in the URL and local storage for the all cases list [#148549](https://github.com/elastic/kibana/pull/148549)
- Sorting by status, severity, and title is now allowed in the all-cases list [#148193](https://github.com/elastic/kibana/pull/148193)
- See "My recently assigned cases" to the recent cases widget of Security Solution dashboard [#147763](https://github.com/elastic/kibana/pull/147763)
- Adds the ability to bulk edit assignees on multiple cases [#146907](https://github.com/elastic/kibana/pull/146907)
Save draft user comment [#146327](https://github.com/elastic/kibana/pull/146327)

Author: Addressed in [902920f](https://github.com/elastic/kibana/pull/157805/commits/902920f25e01eb0671b455f72fc4b4cfab7fddc3).
